llvm/include/llvm/DebugInfo/LogicalView/Core/LVScope.h

```diff
@@ -181,6 +185,7 @@ class LVScope : public LVElement {
   KIND(LVScopeKind, IsTemplatePack);
   KIND_1(LVScopeKind, IsTryBlock, IsBlock);
   KIND_1(LVScopeKind, IsUnion, IsAggregate);
+  KIND_3(LVScopeKind, IsModule, CanHaveRanges, CanHaveLines, TransformName);
```

Comment:
Any specific reason to set `TransformName` bit?

Reply:
No, no particular reason; just a left-over.  Removed.